includes/validation/class-amp-validated-url-post-type.php

```diff
@@ -479,17 +480,7 @@ public static function add_admin_menu_new_invalid_url_count() {
 			return;
 		}
 
-		$new_validation_error_urls = get_transient( static::NEW_VALIDATION_ERROR_URLS_COUNT_TRANSIENT );
-
-		if ( false === $new_validation_error_urls ) {
-			$new_validation_error_urls = static::get_validation_error_urls_count();
-			set_transient( static::NEW_VALIDATION_ERROR_URLS_COUNT_TRANSIENT, $new_validation_error_urls, DAY_IN_SECONDS );
-		} else {
-			// Handle case where integer stored in transient gets returned as string when persistent object cache is not
-			// used. This is due to wp_options.option_value being a string.
-			$new_validation_error_urls = (int) $new_validation_error_urls;
-		}
-
+		$new_validation_error_urls = static::get_validation_error_urls_count();
```

Comment:
Should this instead be `self::` (although the class is not marked as `final`)?

```suggestion
		$new_validation_error_urls = self::get_validation_error_urls_count();
```

Reply:
Just wondering after seeing the alternating uses of `static` and `self` within the class.

Reply:
I think it's in part because we used to support PHP 5.2 in which `static::` didn't exist.

Reply:
Ah OK, that can be attended to at a later time.